src/lib/mina_lib/mina_lib.ml

```diff
@@ -965,9 +965,16 @@ let create ?wallets (config : Config.t) =
                     Trust_system.Peer_trust.peer_statuses config.trust_system
                   in
                   let git_commit = Coda_version.commit_id_short in
-                  let uptime =
-                    Time.diff (Time.now ()) config.start_time
-                    |> Time.Span.to_string_hum ~decimals:1
+                  let uptime_minutes =
+                    let now = Time.now () in
+                    let minutes_float =
+                      Time.diff now config.start_time |> Time.Span.to_min
+                    in
+                    (* if rounding fails, just convert *)
+                    Option.value_map
+                      (Float.iround_nearest minutes_float)
```

Comment:
Should this be `iround_down` if we want to capture 0-5 mins, 5-10 mins, etc.?

Reply:
Should 5.99 minutes be considered 5 minutes, for the purpose of bucketing?

Though I don't think it's critical, either way.